### PR TITLE
Enhancements for the Authentication mechanisms documentation

### PR DESCRIPTION
**Pull request by MichalMaler**
QDOCS-93: Authentication mechanisms revamping

* Fixing the names to match diataxes style
* Adding a missing ID
* Cleaning content and moving it to dedicated sections
* Adding tables for the complete overview
* Style/grammar fixes

Initial preview for tables movements:
https://github.com/quarkusio/quarkus/blob/a3d55ddf0f11b353d0835e76760de5fe9e2ef75a/docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc

Renaming/Redirecting PR for these changes:
https://github.com/quarkusio/quarkusio.github.io/pull/1623

Signed-off-by: Michal Maléř <mmaler@redhat.com>

### CODE REVIEW

**Comment by MichalMaler** [comment]
@sberyozkin @jmartisk Hello Sergey and Jan! Here are some changes for the mentioned section. It is possible that we will need to move the revamped tables to a more specific sections, and I will provide a final review when this shifts will be done.

Xrefs are currently not tested yet. Will do it as a final step

Can I ask you for a review?

Thank You.

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc [hunk 1]
```diff
@@ -205,9 +207,17 @@ For more information, see xref:security-oauth2.adoc[Using OAuth2].
 
 [[oidc-jwt-oauth2-comparison]]
 
-== Choosing between OpenID Connect, SmallRye JWT, and OAuth2 authentication mechanisms
+== Choose the authentication mechanism
+
+Quarkus provides the following authentication mechanisms to secure your Quarkus applications:
```
Comment (jmartisk) [inline docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:212]: This IMO makes it look like it is an exhaustive list (and that Quarkus doesn't support anything beyond these 3), which isn't true. It's just that this section attempts to explain the differences between these 3 mechanisms in particular, but there are more (for example, just a hardcoded list of users and passwords).

Comment (MichalMaler) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:212]: Hm, ok. Will put it back. Thank you

File: docs/src/main/asciidoc/security-overview-concept.adoc [hunk 2]
```diff
@@ -48,7 +48,7 @@ Guidance for testing Quarkus Security features and ensuring that your Quarkus ap
 === Cross-origin resource sharing
 
 To make your Quarkus application accessible to another application running on a different domain, you need to configure cross-origin resource sharing (CORS).
-For more information about the CORS filter that is provided by Quarkus, see the xref:http-reference.adoc#cors-filter[HTTP reference] information.
+For more information about the CORS filter that Quarkus provide, see the xref:http-reference.adoc#cors-filter[HTTP reference] information.
```
Comment (jmartisk) [inline docs/src/main/asciidoc/security-overview-concept.adoc:51]: provides

Comment (jmartisk) [reply docs/src/main/asciidoc/security-overview-concept.adoc:51]: Also you repeat the word "information" twice in this sentence

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc [hunk 3]
```diff
@@ -235,71 +241,55 @@ While `quarkus-oidc` and `quarkus-elytron-security-oauth2` can also verify such
...
-JWT tokens are significantly longer than the opaque tokens but the providers are effectively delegating storing most of the token-associated state to the client by storing it as the token claims and either signing or encrypting them.
+JWT tokens are significantly longer than opaque tokens. Still, the providers effectively delegate most of the token-associated state to the client by storing it as the token claims and either signing or encrypting them.
 ====
 
-The following table provides a summary of the options for each authentication mechanism:
+.Credential types and authentication mechanisms
+|===
+^|Credential type ^| Authentication ^| Notes
+
+^|Username and password ^|xref:security-basic-authentication-concept.adoc[Basic ], xref:security-basic-authentication-concept.adoc#form-auth[Form], xref:security-oidc-bearer-authentication-concept.adoc[OIDC Bearer] ^|
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:254]:
```suggestion
^|Username and password ^|xref:security-basic-authentication-concept.adoc[Basic], xref:security-basic-authentication-concept.adoc#form-auth[Form] ^|
```

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc [hunk 4]
```diff
@@ -235,71 +241,55 @@ While `quarkus-oidc` and `quarkus-elytron-security-oauth2` can also verify such
...
+
+^|Bearer Access Token  ^| xref:security-oidc-bearer-authentication-concept.adoc[OIDC Bearer], xref:security-jwt.adoc[JWT], xref:security-oauth2.adoc[OAuth2]  ^|See the <<table>> table below.
+
+^|Single Sign-On (SSO) ^|xref:security-oidc-bearer-authentication-concept.adoc[OIDC Bearer], xref:security-basic-authentication-concept.adoc#form-auth[Form]   ^|
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:258]: It is a built-in authentication where the form auth is described

Comment (sberyozkin) [inline docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:258]: We should probably rename the `Credential Type` header to `Authentication Requirement` (and the next one to `Authentication Mechanism`) as SSO is not really a type and for ex Keycloak can use many forms of authentication in the SSO flows (I know I proposed to list it here :-) ).
Also the OIDC link should link to the oidc web authentication concepr

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc [hunk 5]
```diff
@@ -235,71 +241,55 @@ While `quarkus-oidc` and `quarkus-elytron-security-oauth2` can also verify such
...
-|No
-|With injected JWT parser
-|No
-|Accept JWT as a cookie
-|No
-|Yes
-|No
+
+[[table]]
+.Quarkus authentication extensions summary
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:268]:
```suggestion
.Token authentication mechanism features
```

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc [hunk 6]
```diff
@@ -235,71 +241,55 @@ While `quarkus-oidc` and `quarkus-elytron-security-oauth2` can also verify such
...
-Your decision to choose whether to use opaque or JWT token format will be driven by architectural considerations.
+Architectural considerations drive your decision to use opaque or JWT token format.
 Opaque tokens tend to be much shorter than JWT tokens but need most of the token-associated state to be maintained in the provider database.
 Opaque tokens are effectively database pointers.
-JWT tokens are significantly longer than the opaque tokens but the providers are effectively delegating storing most of the token-associated state to the client by storing it as the token claims and either signing or encrypting them.
+JWT tokens are significantly longer than opaque tokens. Still, the providers effectively delegate most of the token-associated state to the client by storing it as the token claims and either signing or encrypting them.
 ====
 
-The following table provides a summary of the options for each authentication mechanism:
+.Credential types and authentication mechanisms
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:250]: This table is a higher level table which should not be placed here in the section comparing OIDC/JWT/OAuth2, 
I'd consider placing at the very top of the doc just above Built in mechanisms. Or may be in the main security concept

**Review by sberyozkin — COMMENTED** [review]
@MichalMaler Thanks for going ahead with creating a new table, IMHO it would be useful, a left a few comments.
By the way, please tweak the mutual TLS link at the very top of the builtin mechanisms section. I'm not sure though we need those Basic/Form/MTLS links given that right below them Basic/Form/MTLS subsections start ? Michelle, @michelle-purcell would you like to retain them ?

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc (hunk 4 above)
Comment (MichalMaler) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:258]: OK, renaming. :)

Comment (MichalMaler) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:258]: So, do you want me to remove the SSO row completly, or replace SSO with the Built-in Aut?
@sberyozkin

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc (hunk 6 above)
Comment (MichalMaler) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:250]: Moving it to the top for now. Will fix the rest and let you know. Thank you!

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc (hunk 4 above)
Comment (jmartisk) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:258]: I thought the goal of this table was to purely list credential types (password, certificate, token), and that "authentication requirements" are rather covered in the existing table (by that I mean token verification, single-sign on, multi-tenancy,...), no?

**Comment by MichalMaler** [comment]
> @MichalMaler Thanks for going ahead with creating a new table, IMHO it would be useful, a left a few comments. By the way, please tweak the mutual TLS link at the very top of the builtin mechanisms section. I'm not sure though we need those Basic/Form/MTLS links given that right below them Basic/Form/MTLS subsections start ? Michelle, @michelle-purcell would you like to retain them ?

@sberyozkin I fixed the link. Will remove them if @michelle-purcell agrees they are redundant or just residuals after the content movement :)  Let me know.

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc (hunk 4 above)
Comment (sberyozkin) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:258]: @MichalMaler No, I meant that the form link is wrong, points to the basic auth

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc [hunk 7]
```diff
@@ -207,7 +209,9 @@ For more information, see xref:security-oauth2.adoc[Using OAuth2].
 
 == Choosing between OpenID Connect, SmallRye JWT, and OAuth2 authentication mechanisms
 
-Use the following information to help you to decide which authentication mechanism to use to secure your Quarkus applications:
+Use the following information to select the proper authentication mechanism for your applications.
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:212]:
```suggestion
Use the following information to select the proper token authentication mechanism for your applications.
```

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc (hunk 4 above)
Comment (MichalMaler) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc]: Fixing. My bad.

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc [hunk 8]
```diff
@@ -3,19 +3,36 @@
...
 
 Before you choose an authentication mechanism for securing your Quarkus applications, review the information provided.
 
+.Authentication requirements and mechanisms
+|===
+^|Authentication Requirement ^| Authentication Mechanism ^| Notes
+
+^|Username and password ^|xref:security-basic-authentication-concept.adoc[Basic], xref:security-authentication-mechanisms-concept.adoc#form-auth[Form] ^|
+
+^|Bearer Access Token  ^| xref:xref:security-openid-connect-web-authentication.adoc[OIDC], xref:security-jwt.adoc[JWT], xref:security-oauth2.adoc[OAuth2]  ^|See the <<table>> table below.
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:18]: This link should point to the bearer access token doc, currently named `security-openid-connect.adoc`

Comment (sberyozkin) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:18]: I see, double `xref` here

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc [hunk 9]
```diff
@@ -3,19 +3,36 @@
...
+
+^|Single Sign-On (SSO) ^|xref:xref:security-openid-connect-web-authentication.adoc[OIDC], xref:security-authentication-mechanisms-concept.adoc#form-auth[Form]   ^|
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:20]:
```suggestion
^|Single Sign-On (SSO) ^|xref:security-openid-connect-web-authentication.adoc[OIDC], xref:security-authentication-mechanisms-concept.adoc#form-auth[Form]   ^|
```

Comment (MichalMaler) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:20]: nice catch

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc (hunk 8 above)
Comment (MichalMaler) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:18]: Fixing

Comment (sberyozkin) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:18]: @MichalMaler The other thing is that `Notes` column seems redundant right now, it was my idea I know, but it does not look good yet while we only have a single note, as it creates a widest column, with all but only a single value set, my proposal is to collapse that note into the 2nd column `OIDC, JWT, OAuth2 (see <<table>> table below)`

Comment (MichalMaler) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:18]: Yeah, I already tried that. The problem is that it doenst look well and asciidoc doesnt allowed me to use bullet point for that Table-note. Need to find a way how to do it. Working on it

Comment (MichalMaler) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:18]: Nice, found a solution. Adding.

Comment (MichalMaler) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc]: @sberyozkin by that file `security-openid-connect.adoc`, did you meant `security-openid-connect-client.adoc` ?

Comment (MichalMaler) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc]: it was this one I guess `security-oidc-bearer-authentication-concept.adoc` Renamed and recomposed by Rolfe.

**Comment by MichalMaler** [comment]
@sberyozkin I think we are ready to merge.
The `xref:security-oidc-code-flow-authentication-concept.adoc[OIDC Web]` link will be working when the Sheila's PR (https://github.com/quarkusio/quarkus/pull/30518) is merged.

**Comment by MichalMaler** [comment]
@sberyozkin This will be needed to be merged too:
https://github.com/quarkusio/quarkusio.github.io/pull/1623
Renaming of the webatun file we are pointing to in our table

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc [hunk 10]
```diff
@@ -217,14 +241,16 @@ In both cases, `quarkus-oidc` requires a connection to the specified OpenID Conn
 
 * If your Bearer tokens must be verified, use `quarkus-oidc`, `quarkus-smallrye-jwt`, or `quarkus-elytron-security-oauth2`.
 
-* If your Bearer tokens are in a JWT format, you can use either of the three extensions. Both `quarkus-oidc` and `quarkus-smallrye-jwt` support refreshing the JsonWebKey (JWK) set when the OpenID Connect provider rotates the keys.
+* If your Bearer tokens are in a JWT format, you can use either of the three extensions.
```
Comment (jherrman) [inline docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:244]:
```suggestion
* If your Bearer tokens are in a JWT format, you can use any of the extensions listed above.
```

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc [hunk 11]
```diff
@@ -217,14 +241,16 @@ In both cases, `quarkus-oidc` requires a connection to the specified OpenID Conn
...
 Therefore, if remote token introspection must be avoided or is unsupported by the providers, use `quarkus-oidc` or `quarkus-smallrye-jwt` for verifying JWT tokens.
 
 * If you need to introspect the JWT tokens remotely, you can use either `quarkus-oidc` or `quarkus-elytron-security-oauth2` because they support the verification of the opaque or binary tokens by using remote introspection.
 `quarkus-smallrye-jwt` does not support the remote introspection of both opaque or JWT tokens but instead relies on the locally available keys that are usually retrieved from the OpenID Connect provider.
 
-* `quarkus-oidc` and `quarkus-smallrye-jwt` support the injecting of JWT and opaque tokens into the endpoint code.
-Injected JWT tokens provide more information about the user. All extensions can have the tokens injected as `Principal`.
+* `quarkus-oidc` and `quarkus-smallrye-jwt` support the JWT and opaque tokens injection into the endpoint code.
+Injected JWT tokens provide more information about the user.
+All extensions can have the tokens injected as `Principal`.
```
Comment (jherrman) [inline docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:253]: Will the target audience know what this means?

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc [hunk 12]
```diff
@@ -235,71 +261,38 @@ While `quarkus-oidc` and `quarkus-elytron-security-oauth2` can also verify such
 
 [NOTE]
 ====
-Your decision to choose whether to use opaque or JWT token format will be driven by architectural considerations.
+Architectural considerations drive your decision to use opaque or JWT token format.
```
Comment (jherrman) [inline docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:264]:
```suggestion
To decide whether to use opaque or JWT token format, consider your hardware architecture.
```

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc [hunk 13]
```diff
@@ -235,71 +261,38 @@ While `quarkus-oidc` and `quarkus-elytron-security-oauth2` can also verify such
...
 Opaque tokens tend to be much shorter than JWT tokens but need most of the token-associated state to be maintained in the provider database.
 Opaque tokens are effectively database pointers.
-JWT tokens are significantly longer than the opaque tokens but the providers are effectively delegating storing most of the token-associated state to the client by storing it as the token claims and either signing or encrypting them.
+JWT tokens are significantly longer than opaque tokens. Still, the providers effectively delegate most of the token-associated state to the client by storing it as the token claims and either signing or encrypting them.
```
Comment (jherrman) [inline docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:267]:
```suggestion
JWT tokens are significantly longer than opaque tokens. However, the providers delegate most of the token-associated state to the client by storing it as the token claims and either signing or encrypting them.
```

File: docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc [hunk 14]
```diff
@@ -4,7 +4,7 @@ and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
 [id="security-authorize-web-endpoints-reference"]
-= Authorize web endpoints
+= Authorization of web endpoints
```
Comment (jherrman) [inline docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc:7]:
```suggestion
= Authorizing web endpoints
```

Comment (MichalMaler) [reply docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc:7]: The above heading format is good regarding the Diataxes reference format we are using in this repo. But thank you so much for the rest of the suggestions! :) Will add for sure!

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc (hunk 11 above)
Comment (MichalMaler) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:253]: I guess they will :) Thank you for the rev!

**Comment by sberyozkin** [comment]
Thanks @MichalMaler Waiting for Sheila to update her PR as we agreed in the offline discussion, that PR is nearly ready go, a few minor updates are needed

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc [hunk 15]
```diff
@@ -207,7 +229,9 @@ For more information, see xref:security-oauth2.adoc[Using OAuth2].
 
 == Choosing between OpenID Connect, SmallRye JWT, and OAuth2 authentication mechanisms
 
-Use the following information to help you to decide which authentication mechanism to use to secure your Quarkus applications:
+Use the following information to select the proper token authentication mechanism for your applications.
```
Comment (sheilamjones) [inline docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:232]: Style suggestion: I think the use of "proper" here could be a bit ambiguous as in, would a user know which is proper mechanism for their application and which is now? I think "appropriate" might be better.
Nitpick: For maximum clarity, I think I would keep the previous wording for the end of the sentence, that is,  "Use the following information to select the appropriate token authentication mechanism to secure your Quarkus applications."

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc [hunk 16]
```diff
@@ -235,71 +261,38 @@ While `quarkus-oidc` and `quarkus-elytron-security-oauth2` can also verify such
...
+^|Feature required 3+^| Authentication mechanism
+
+^| ^s|`quarkus-oidc` ^s|`quarkus-smallrye-jwt` ^s| `quarkus-elytron-security-oauth2`
+
+s|Bearer JWT verification ^|Local verification or introspection  ^|Local verification  ^|Introspection
+
+s|Bearer opaque token verification ^|Introspection   ^|No  ^|Introspection
+s|Refreshing `JsonWebKey` set for verifying JWT tokens  ^|Yes   ^|Yes  ^|No
+s|Represent token as `Principal`  ^|Yes   ^|Yes  ^|Yes
+s|Inject JWT as MP JSON Web Token (JWT)  ^|Yes   ^|Yes  ^|No
```
Comment (sheilamjones) [inline docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:282]: Style/Capitalization: 
- Use sentence case - "Inject JWT as MP JSON web token 
- Is it necessary to have (JWT) written in brackets in this instance given that it is already used in a few previous instances in the table. Would it be possible to instead place it after the very first occurrence in the topic?

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc [hunk 17]
```diff
@@ -235,71 +261,38 @@ While `quarkus-oidc` and `quarkus-elytron-security-oauth2` can also verify such
...
+
+s|Authorization Code Flow  ^| Yes  ^|No  ^|No
```
Comment (sheilamjones) [inline docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:284]: Capitalization: propose to change to "Authorization code flow"

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc [hunk 18]
```diff
@@ -235,71 +261,38 @@ While `quarkus-oidc` and `quarkus-elytron-security-oauth2` can also verify such
...
+s|Multi-tenancy ^| Yes  ^|No  ^|No
+s|User information support  ^| Yes  ^|No  ^|No
+s|PEM key format support  ^|No   ^|Yes  ^|No
```
Comment (sheilamjones) [inline docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:287]: Should "PEM" be written out in full here for this first instance?

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc [hunk 19]
```diff
@@ -3,19 +3,39 @@
...
+
 == Built-in authentication support
 
 Quarkus Security provides the following built-in authentication mechanisms:
 
 * xref:security-basic-authentication-concept.adoc[Basic authentication]
-* <<Form-based authentication>>
-* <<mutual TLS authentication>> 
+* <<form-auth>>
+* <<mutual-tls>>
```
Comment (sheilamjones) [inline docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:38]: In the rendered preview, there are brackets "[ ]" appearing around the this "mutual-tls" link. Just want to double-check that is rendering correctly from your side.

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc [hunk 20]
```diff
@@ -3,19 +3,39 @@
...
+.Authentication requirements and mechanisms
+[source, adoc]
+|===
+^|Authentication Requirement ^| Authentication Mechanism
+
+^|Username and password ^|xref:security-basic-authentication-concept.adoc[Basic], xref:security-authentication-mechanisms-concept.adoc#form-auth[Form]
+
+^|Bearer Access Token  ^a| xref:security-oidc-bearer-authentication-concept.adoc[OIDC Bearer], xref:security-jwt.adoc[JWT], xref:security-oauth2.adoc[OAuth2]
+
+* See the <<table>> table below.
```
Comment (sheilamjones) [inline docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:21]: In the renderend output, this is the only table entry that has also has an entry in the Notes column. When this column is blank for all the other items in the table, it looks a bit funny and also might pose an issue for Accessibility.  Would it be possible to remove the Notes column and move this one item underneath the table (perhaps as a callout)

**Comment by sheilamjones** [comment]
Hi @MichalMaler, good job on the restructuring here. I just had a couple of minor comments and suggestions. 
Kind regards,
Sheila

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc (hunk 16 above)
Comment (sberyozkin) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:282]: `JSON Web Token` is how it is used on the web/everywhere, it is OK IMHO

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc (hunk 19 above)
Comment (sberyozkin) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:38]: I saw it as well

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc (hunk 20 above)
Comment (MichalMaler) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:21]: Hello Sheila @sheilamjones . Rendered output was just for content moving. It is very obsolete right now. You can have a new rendered view when you go: Files changed -> then in the right upper corner -> ... -> View file.
Please, always use the last one.
Adding it here:
https://github.com/quarkusio/quarkus/blob/d0f419683041593c0a792311fdf37149083833a3/docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc (hunk 19 above)
Comment (MichalMaler) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:38]: It renders correctly in the latest version of a preview.

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc (hunk 15 above)
Comment (MichalMaler) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:232]: OK, will apply.

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc (hunk 16 above)
Comment (MichalMaler) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:282]: Will do. Thx for the catch

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc (hunk 18 above)
Comment (MichalMaler) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:287]: OK, will add.

**Comment by MichalMaler** [comment]
> Hi @MichalMaler, good job on the restructuring here. I just had a couple of minor comments and suggestions. Kind regards, Sheila

Thank you for quick and great review, as always :) Applied most of your suggestions. GJ! @sheilamjones

**Comment by MichalMaler** [comment]
GH Preview: https://github.com/quarkusio/quarkus/blob/d0f419683041593c0a792311fdf37149083833a3/docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc [hunk 21]
```diff
@@ -3,19 +3,39 @@
...
+
+^|Single Sign-On (SSO) ^|xref:security-oidc-code-flow-authentication-concept.adoc[OIDC Web], xref:security-authentication-mechanisms-concept.adoc#form-auth[Form]
+
+^|Client Certificate  ^|xref:security-authentication-mechanisms-concept.adoc#mutual-tls[Mutual TLS (MTLS)]
+
+^|WebAuthn ^|xref:security-webauthn-concept.adoc[WebAuthn]
+
+^|Kerberos ticket    ^|link:https://quarkiverse.github.io/quarkiverse-docs/quarkus-kerberos/dev/index.html[Quarkus - Kerberos]
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:29]:
```suggestion
^|Kerberos ticket    ^|link:https://quarkiverse.github.io/quarkiverse-docs/quarkus-kerberos/dev/index.html[Kerberos]
```

Comment (sberyozkin) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:29]: These are all Quarkus extensions, so propose to remove `Quarkus -`. This is the last change proposal from me, thanks @MichalMaler

**Review by sberyozkin — APPROVED** [review]
IMHO it is now nearly ready to go.

@sheilamjones Please update your PR when you get a chance as Michal's PR points to the file which will only be resolvable once your PR gets merged :-), so for now please don't merge

File: docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc (hunk 20 above)
Comment (MichalMaler) [reply docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc:21]: @sberyozkin @sheilamjones Hello. Please, where exactly in the latest build, which is the only relative built when providing a review, do you see that extra Notes column? :(
image

**Comment by MichalMaler** [comment]
Pushed the latest update:
image

**Comment by sberyozkin** [comment]
Hey @MichalMaler Sheila's PR is now on the main branch, can you please rebase and check the links, FYI, the OIDC web app concept doc has been renamed, so some minor sync will me needed for your PR, thanks

**Comment by MichalMaler** [comment]
@sberyozkin Hello Sergey! Thx for letting me know. One additional xref fixed. The whole PR now using the new names that  reflects Sheila's work.
We did some good job here again. Cheers!

**Comment by MichalMaler** [comment]
This should be merged too, but it is not my call:
https://github.com/quarkusio/quarkusio.github.io/pull/1623

**Comment by sberyozkin** [comment]
@MichalMaler Great work indeed, thanks very much

**Comment by gsmet** [comment]
I'm unmarking all the doc PRs for backports. They are all part of massive changes that shouldn't be backported. Please don't mark for backport massive structural changes to the doc.

**Pull request merged**